ClaseForms/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Formularios </title>
+</head>
+<body>
+    <form action="">
+        <label for="nombre">
+            <span>Nombre completo</span>
+            <input type="text" id="Nombre" placeholder="nombre"/>
```

Comment:
Agregaste un label para una etiqueta pero no esta indicado en el input
```suggestion
            <input type="text" id="nombre" name="nombre" placeholder="nombre"/>
```